utils/i18nUtils.ts

```diff
@@ -28,7 +29,7 @@ export const getLinksLanguageAlternative = (
         rel: 'alternate',
         href: getFullPathWihLocale(locale.code),
         hreflang: locale.iso,
-      })
+      } as MetaInfo)
```

Comment:
```suggestion
      })
```
importを直すとここが無くても通るはずです